### PR TITLE
Gradle migration

### PR DESCRIPTION
**Pull request by sarahlensing**

### CODE REVIEW

File: app/src/main/java/com/mapzen/speakerbox/example/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.mapzen.speakerbox.example;
+
+import android.os.Bundle;
+import android.support.design.widget.FloatingActionButton;
```
Comment (ecgreb) [inline app/src/main/java/com/mapzen/speakerbox/example/MainActivity.java]: Some unused imports in here.

File: app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    }
+  }
+}
+
+dependencies {
+  compile project(':library')
+  compile 'com.android.support:appcompat-v7:23.2.0'
+  compile 'com.android.support:design:23.2.0'
+  testCompile 'junit:junit:4.12'
+}
```
Comment (ecgreb) [inline app/build.gradle]: Newline

**Comment by ecgreb** [comment]
Changes look great... but can we get back the old launcher icon ;)

File: app/src/main/java/com/mapzen/speakerbox/example/MainActivity.java (hunk 1 above)
Comment (sarahlensing) [inline app/src/main/java/com/mapzen/speakerbox/example/MainActivity.java]: Removed

File: app/build.gradle (hunk 2 above)
Comment (sarahlensing) [inline app/build.gradle]: Fixed

**Comment by sarahlensing** [comment]
You got it :+1:

**Pull request merged**